Makefile

```diff
@@ -323,7 +323,7 @@ gofmt:
 
 govet:
 	@$(ECHO_CHECK) vetting all GOFILES...
-	$(QUIET)$(GO) vet \
+	$(QUIET) $(GOLIST) $(GO) vet \
```

Comment:
This change appears unrelated?

Reply:
Can you please specify why? I'm not doubting you, but context would be helpful here.

Reply:
My fault, I've updated the commit description.